target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -60,6 +60,11 @@ qcom,ipq8064-db149)
 	ucidef_add_switch "switch0" \
 		"1:lan" "2:lan" "3:lan" "4:lan" "6u@eth1" "5:wan" "0u@eth0"
 	;;
+ubnt,unifi-ap-hd)
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
+	ucidef_set_interface_macaddr "lan" "$(mtd_get_mac_binary EEPROM 0x0)"
+	ucidef_set_interface_macaddr "wan" "$(mtd_get_mac_binary EEPROM 0x6)"
```

Comment:
Why not use DTS for setting the MAC addresses?

Reply:
Missing knowledge of how it's done. :) I'll see if I can figure it out.